DataFormats/BTauReco/interface/TaggingVariable.h

```diff
@@ -78,7 +78,9 @@ namespace reco {
 
       vertexEnergyRatio,                        // ratio of energy at secondary vertex over total energy
       vertexJetDeltaR,                          // pseudoangular distance between jet axis and secondary vertex direction
-
+      
+      flightDistance1dVal,                      // Linear distance between primary and secondary vertex
```

Comment:
"linear distance" is probably not the best description. Maybe "longitudinal distance" or "longitudinal distance along the z axis"...